source/docs/articles/drupal/major-version-drupal-upgrades.md

```diff
@@ -6,26 +6,42 @@ category:
 keywords: drupal, upgrade, upgrading, revision upgrade, update, updating
 ---
-The best practice for doing a major Drupal revision upgrade (e.g. version 6 to version 7) is to start a new site. Even the simplest of upgrades require their own QA and deployment process, and trying to do an upgrade on an existing site is not a recipe for success.
+The best practice for doing a major Drupal revision upgrade (i.e. version 6 to version 7) is to start a new site. Even the simplest of upgrades require their own QA and deployment process, and trying to do an upgrade on an existing site is not a recipe for success.
 
 Also, Pantheon needs to track the proper upstream Git history for your site to deliver core updates. By starting a new site for the upgrade, you ensure that future core updates will be available via the Dashboard.
 
-## Set up the Code and Migrate the Data
+The best way to think about upgrading to the next major revision of Drupal on Pantheon is the same as when you get a new laptop or smartphone: first you set up both devices side-by-side, then you migrate the data from the old to the new.
+
 <div class="alert alert-danger" role="alert">
 <h4>Warning</h4>
 Do not attempt the upgrades on the platform. This is not supported.</div>
 
-The best way to think about upgrading to the next major revision of Drupal on Pantheon is the same as when you get a new laptop or smartphone: first you set up both devices side-by-side, then you migrate the data from the old to the new.
+## Upgrade from Drupal 6 to Drupal 7
 
-For instance, if you are upgrading from Drupal 6 to Drupal 7, follow these steps:
+### Set up the Code and Migrate the Data
 
-1. Start a new site using D7 as the start state.
+1. Start a new site using Drupal 7 as the start state.
 2. Add the 7.x version of your contrib modules.
 3. Import your existing data.
 4. Debug, QA, release.
 
-## Content and Configuration
+### Content and Configuration
 
 While you can try to get Drupal to handle all the data architecture changes between major revisions (importing the old database and running update.php), this is often not a complete solution. Depending on the specific module stack and configuration of your current site, it may be faster and more direct to plan and execute a content migration to the new site rather than trying to use the built-in update tools.
 
 If you are not having much luck with update.php, consider setting up the new site and using tools like the migrate.module to import your existing content. While this might initially seem like more work, it can often lead to a cleaner result more quickly, especially if your new site includes major architectural changes, features, or a redesign.
+
+## Upgrade to Drupal 8
+
+In Drupal 8, the migrate module is now in core and no longer supports upgrading through update.php.
+
+1. Start a new site using Drupal 8 as the start state.
+2. Add the 8.x version of your contrib modules.
+3. Import your existing data.
+4. Debug, QA, release.
+
```

Comment:
Suggest to expand instructions, either within this doc or link to the proposed doc in #923 when it has been created. 

> Import your existing data. 

We should note how to create the imports, and the 2 options for importing. May be worth generating some screenshots for the new Migrate API. See [Upgrading from Drupal 6 or 7 to Drupal 8](https://www.drupal.org/upgrade/migrate).